exodus/search/locale/fr/LC_MESSAGES/django.po

```diff
@@ -19,8 +19,29 @@ msgstr ""
 "Plural-Forms: nplurals=2; plural=(n > 1);\n"
 
 #: templates/search_form.html:6
-msgid "Search"
-msgstr "Recherche"
+msgid "Search a report"
+msgstr "Rechercher un rapport"
+
+msgid "Application name"
+msgstr "Nom de l'application"
+
+msgid "Let's go!"
+msgstr "C'est parti!"
```

Comment:
I think that it has to be `C'est parti !` with the space :)

Reply:
Nice catch, thanks! I'll change it :)